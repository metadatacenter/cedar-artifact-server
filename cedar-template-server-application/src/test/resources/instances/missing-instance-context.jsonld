{
  "sampleName": {
    "@value": "Glen Peewter"
  },
  "sex": {
    "@id": "http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C20197",
    "_valueLabel": "Male"
  },
  "tissue": {
    "@id": "http://purl.obolibrary.org/obo/UBERON_0004228",
    "_valueLabel": "urinary bladder smooth muscle"
  },
  "optionalAttribute": [
    {
      "@context": {
        "pav": "http://purl.org/pav/",
        "oslc": "http://open-services.net/ns/core#",
        "name": "https://schema.metadatacenter.orgx/properties/name",
        "value": "https://schema.metadatacenter.orgx/properties/value"
      },
      "name": {
        "@value": "temperature"
      },
      "value": {
        "@value": "warm"
      }
    }
  ],
  "schema:isBasedOn": "https://repo.metadatacenter.orgx/templates/dc5a0562-6fd8-4cdc-9780-865ba89eeb68",
  "schema:name": "Nested-Field Template metadata",
  "schema:description": "Description",
  "pav:createdOn": "2017-04-12T16:50:18-0700",
  "pav:createdBy": "https://metadatacenter.org/users/67a894dc-e9df-42d0-bb6e-5dbb4aa9dd79",
  "pav:lastUpdatedOn": "2017-04-12T16:50:18-0700",
  "oslc:modifiedBy": "https://metadatacenter.org/users/67a894dc-e9df-42d0-bb6e-5dbb4aa9dd79",
  "@id": "https://repo.metadatacenter.orgx/template-instances/977a3140-f6c0-4f12-859a-46110433224d"
}