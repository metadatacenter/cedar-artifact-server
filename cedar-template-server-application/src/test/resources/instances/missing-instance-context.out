{"validates":"false","warnings":["the following keywords are unknown and will be ignored: [@context, @id, @type, _ui, oslc:modifiedBy, pav:createdBy, pav:createdOn, pav:lastUpdatedOn]","the following keywords are unknown and will be ignored: [@context, @id, @type, _ui, oslc:modifiedBy, pav:createdBy, pav:createdOn, pav:lastUpdatedOn]","the following keywords are unknown and will be ignored: [@context, @id, @type, _ui, _valueConstraints, oslc:modifiedBy, pav:createdBy, pav:createdOn, pav:lastUpdatedOn]","the following keywords are unknown and will be ignored: [@context, @id, @type, _ui, _valueConstraints, oslc:modifiedBy, pav:createdBy, pav:createdOn, pav:lastUpdatedOn]","the following keywords are unknown and will be ignored: [@context, @id, @type, _ui, _valueConstraints, oslc:modifiedBy, pav:createdBy, pav:createdOn, pav:lastUpdatedOn]","the following keywords are unknown and will be ignored: [@context, @id, @type, _ui, _valueConstraints, oslc:modifiedBy, pav:createdBy, pav:createdOn, pav:lastUpdatedOn]","the following keywords are unknown and will be ignored: [@context, @id, @type, _ui, _valueConstraints, oslc:modifiedBy, pav:createdBy, pav:createdOn, pav:lastUpdatedOn]"],"errors":["object has missing required properties ([\"@context\"])"]}